### PR TITLE
makes powerfist replace your arm

### PR DESCRIPTION
**Pull request by sylvia-from-fulp-station**
## About The Pull Request
JIYU7U22QS
ezgif-1-9426e330a6

The power fist costs 10 tc, and cannot be dropped without removing your arm.
ezgif-1-790b9e37dc

ZJmYoTkQoG
The desword description is different
~~You can now get an energy sword at round start.~~
## Why It's Good For The Game
I noticed alot of interesting things looking at the most used melee weapons.
At the top of our overused weapons lists sits the circular saw; with its robustness, reliability and wide availability it is easy to see why it tops the charts. Below are various tools; but each pail in one category compared to the circular saw. Except perhaps the fire axe; which is uniquely the most used melee weapon on terry and has been seen every round there; despite only two existing on station.
Next is the baton and kitchen knife; which interestingly sit at almost exactly the same place, despite neither being truly the best option in their own category
vivaldi_xuTW87grb1
In the middle is the good old toolbox; it relies on rng knockdowns but can still be just as effective if you are lucky; and is easier to get than a saw.
vivaldi_EbJPxKiW52
Now showing up in only 1/4th of every round; lays the esword. the "most robust melee weapon in the game"
vivaldi_lw9ZZnHosh
Spy has been generating enough katanas that they are close to the energy sword. Lol.

~~Its fucked up a cultist can have a pretend e-sword that can't be used against them before a traitor has a real one.~~
~~After decap removal an esword is not as scary as a bomb; even going out of your way to gut and nugget someone you could simply light them on fire and leave instead. By the time traitors can access it the station is usually swiss cheese already.~~
~~With the esword losing its ability to easily mass round remove people via behead + head hiding~~ the powerfist is the best melee weapon traitors have access to and it's underused except by players who are good enough to kill entire crews with it. ~~I dont know that the cost increase will stop this; but it brings it in line with the other powerful melee weapons.~~ I don't know that the nodrop will stop this, but it gives the powerfist a legitimate downside over other melee weapons while retaining its full usage; also giving the situational advantage of nodrop to soften the blow of the nerf.

## Changelog

:cl:Sylvia
balance: The cost of the power fist has been increased to 10 telecrystals.
balance: Once purchased, or otherwise held the power-fist can no longer be removed. The syndicate will provide no compensation for grievous bodily harm done in attempting to remove these devices.
qol: The dual bladed energy sword's warranty has been updated to reflect that the 100% energy weapon proof guarantee was cancelled.
/:cl:

### CODE REVIEW

**Comment by Dorsisdwarf** [comment]
I'm not sure I quite understand the use of stats here but, it seems that the argument is that free crew-available items are more commonly used as weapons than expensive purchases available to two types of antag? 

I'd be quite alarmed if energy swords were one of the most commonly used weapons in the game given the average round has about 50 people in it, of who maybe 5 at most are able to buy energy swords (But just as equally maybe _none_), but those 5 possible-users might also want to spend their picks on anything else in the big list of stuff, or not at all?

Personally I think a restricted, expensive, extremely loud and obvious weapon, available by choice only to traitors and nuke ops, being used in half as many rounds as toolboxes, telebatons, and the captains ego saber sounds about right?

**Comment by EnterTheJake** [comment]
The powerfist also has no block chance and requires atmo canisters to use, if you are making it cost that much i'd suggest you give him some sort of anti-drop, since you know it's a glove, it's kinda dumb it can be dropped.

**Comment by carlarctg** [comment]
> The powerfist also has no block chance and requires atmo canisters to use, if you are making it cost that much i'd suggest you give him some sort of anti-drop, since you know it's a glove, it's kinda dumb it can be dropped.

The lack of block chance is made up for by the guaranteed stun upon punching the victim into a wall. The atmos canister requirement is made up for by it being able to eject out gasses when hitting (water vapor)

the glove thing makes 'sense' but would make it truly overpowered, so I can't say that's a good idea at all.

**Comment by EnterTheJake** [comment]
> > The powerfist also has no block chance and requires atmo canisters to use, if you are making it cost that much i'd suggest you give him some sort of anti-drop, since you know it's a glove, it's kinda dumb it can be dropped.
> 
> The lack of block chance is made up for by the guaranteed stun upon punching the victim into a wall. The atmos canister requirement is made up for by it being able to eject out gasses when hitting (water vapor)
> 
> the glove thing makes 'sense' but would make it truly overpowered, so I can't say that's a good idea at all.

it is getting bumped to 10 tc, the stun is situational not guaranteed.

We are still in the stun baton meta, a melee weapon without block chance that costs half of your tc can probably have anti drop as an extra treat.

**Comment by Formerxboxgamer** [comment]
> > > The powerfist also has no block chance and requires atmo canisters to use, if you are making it cost that much i'd suggest you give him some sort of anti-drop, since you know it's a glove, it's kinda dumb it can be dropped.
> > 
> > 
> > The lack of block chance is made up for by the guaranteed stun upon punching the victim into a wall. The atmos canister requirement is made up for by it being able to eject out gasses when hitting (water vapor)
> > the glove thing makes 'sense' but would make it truly overpowered, so I can't say that's a good idea at all.
> 
> it is getting bumped to 10 tc, the stun is situational not guaranteed.
> 
> We are still in the stun baton meta, a melee weapon without block chance that costs half of your tc can probably have anti drop as an extra treat.

Nah, power fist is the most powerful melee weapon in the game and its not even close. No one uses it so no one understands how busted it is. 2 shots to crit someone and 3 shots in riot armor with a hardstun.

**Comment by Deadgebert** [comment]
powerfist is pretty busted and a nerf is probably needed. I'm glad it's done with TC cost instead of nerfing the actual item.

**Comment by sylvia-from-fulp-station** [comment]
> The powerfist also has no block chance and requires atmo canisters to use, if you are making it cost that much i'd suggest you give him some sort of anti-drop, since you know it's a glove, it's kinda dumb it can be dropped.

Sure

**Comment by Deadgebert** [comment]
Giving powerfist anti-drop or putting it on glove slot is a huge buff. That's way more valuable than the extra 4 tc cost.

**Comment by sylvia-from-fulp-station** [comment]
> Giving powerfist anti-drop or putting it on glove slot is a _huge_ buff. That's way more valuable than the extra 4 tc cost. If the intention is to nerf it, that's not the way to do it.
I made it an arm replacement in line with chaplain nullrods as a snowflake no drop item sounds cringe.

**Comment by sylvia-from-fulp-station** [comment]
> Giving powerfist anti-drop or putting it on glove slot is a _huge_ buff. That's way more valuable than the extra 4 tc cost. If the intention is to nerf it, that's not the way to do it.

I made it an arm replacement in line with chaplain nullrods as a snowflake no drop item sounds cringe.

**Comment by GPeckman** [comment]
"Hey maybe you should give the powerfist anti drop or something so it's harder to disarm"
_Monkey's paw curls_

Jokes aside, this is a pretty good change. The powerfist is definitely one of the strongest uplink weapons, and could use some tweaking.

**Comment by optimumtact** [comment]
> They are there to show energy sword is underused.

as was just pointed out to you, that is not what it shows.

You need to look at the list of tc purchases instead during rounds.

**Comment by jughu** [comment]
> "Hey maybe you should give the powerfist anti drop or something so it's harder to disarm" _Monkey's paw curls_
> 
> Jokes aside, this is a pretty good change. The powerfist is definitely one of the strongest uplink weapons, and could use some tweaking.

And nobody almost buys it from the uplink this is just spy generating power fists as bounty

**Comment by sylvia-from-fulp-station** [comment]
> You need to look at the list of tc purchases instead during rounds.

It's in the top 5 most purchased TC items; and is bought about 10x more than the power fist.
This is irrelevant. Almost every round generates atleast one traitor and it's seen in 1/4th of rounds; its by no means underused by traitors, but it has so many imitators available that the actual thing is diminished. Ideally this increases the usage, slightly. I don't want it to be in a specific number of rounds because that is a bat shit way to balance a game, i want it to be fun.

**Comment by EnterTheJake** [comment]
People understimate how big of a deal block chance is when 1 click can lose you the fight.

> > Giving powerfist anti-drop or putting it on glove slot is a _huge_ buff. That's way more valuable than the extra 4 tc cost. If the intention is to nerf it, that's not the way to do it.
> 
> I made it an arm replacement in line with chaplain nullrods as a snowflake no drop item sounds cringe.

It still seem unecessary to me, to even touch the item i mean.

"oh but there's like 3 people that  are very good at using" doesn't really hold much meaning to me.

We don't and shouldn't balance the game around the 1% of players.

I just checked the charts for most bought uplink items, PowerFist has such a low pickrate you cannot even find it on both LRP and MRP.

https://superset.moth.fans/superset/dashboard/4/?native_filters_key=QXoguTsU6VknLceRcrX5nRvxudoKwRX9dqiGe21dre__sM4BVjQDnxieA-5LnO7X

You may argue that Stats aren't everything, but you are almost doubling the price of an item that barely sees any play.

Now if you were to make the item more accessible in exchange for the price increase, i could maybe get behind the change that's why i proposed a no drop.

**Comment by sylvia-from-fulp-station** [comment]
> You may argue that Stats aren't everything, but you are almost doubling the price of an item that barely sees any play.

yes

**Comment by Deadgebert** [comment]
> > Giving powerfist anti-drop or putting it on glove slot is a _huge_ buff. That's way more valuable than the extra 4 tc cost. If the intention is to nerf it, that's not the way to do it.
> 
> I made it an arm replacement in line with chaplain nullrods as a snowflake no drop item sounds cringe.

eh. i'm now fully against it. second arm is too important. this is the type of thing i didn't want to see happen. I'd rather just see the damage or gas usage nerfed.

**Comment by saprasam** [comment]
i think the powerfist could use a tc raise but the rest of these changes to it are redundant & would make the item's use drop to a niche rare use-case due to the fact that you WILL get instantly validhunted as soon as somebody sees big grey block on arm that can't be removed

**Comment by sylvia-from-fulp-station** [comment]
>WILL get instantly validhunted as soon as somebody sees big grey block on arm that can't be removed
just kill them with the power fist man

**Comment by optimumtact** [comment]
> > You need to look at the list of tc purchases instead during rounds.
> 
> It's in the top 5 most purchased TC items; and is bought about 10x more than the power fist. This is irrelevant.

Your whole thesis for this change is the thing is underused, and yet it's the 5th most used traitor item

which means the data does not support your assertions and you need to find a new justification.

**Comment by EnterTheJake** [comment]
You should perhaps have a talk to Watermelon about this pr, Timelock was introduced to most traitors lethal items  precisely to avoid roundstart murderboning.

**Comment by sylvia-from-fulp-station** [comment]
> which means the data does not support your assertions and you need to find a new justification.

My thesis touches on multiple things; i looked at the data and it was pretty inconclusive; threw it in because i found it interesting, which it says at the top there. `I noticed a lot of interesting things` not, `these numbers mean i should do this`

The whole paragraph below the data ; doesn't even reference said data. I respect the intelligence of the reader enough to draw their own conclusions from it.

tldr for oranges; its not as scary because it doesn't decapitate and inherently round remove anymore, so i don't think it should be rep locked as if it can cause massive destruction on its own.

**Comment by Potato-Masher** [comment]
I can understand the price change of the power fist, it is quite powerful with proper setup. Though the keywords being "proper setup". To run at it's best the powerfist user has to do a number of tasks to prepare the gas tanks to be able to run at setting 3 with equipment that may not be easily accessible depending on access, not to mention it's an ammo fed melee weapon with it's reload requiring fumbling with tools to remove the spent tank.

Rather than being a nodrop weapon, perhaps just make it bulky? It would achieve a similar result of making the powerfist difficult to conceal, while still allowing the traitor using it to hide and stash it somewhere if needed. Sacrificing a whole arm for a traitor dead giveaway makes it even less desirable to the average user.

**Comment by sylvia-from-fulp-station** [comment]
>Sacrificing a whole arm for a traitor dead giveaway makes it even less desirable to the average user.

This i can understand; however it's already incredibly undesirable to the average user. I'm not trying to make more or less people use it with this change, which is why i've left the core item the same; i want it to have a unique downside, instead of just an incredibly long list of upsides after you've finished a fetch quest.

**Review by Watermelon914 — CHANGES_REQUESTED** [review]
revert esword changes

**Comment by sylvia-from-fulp-station** [comment]
> revert esword changes

jig-jig-is-up

**Comment by Bibarra** [comment]
> > Sacrificing a whole arm for a traitor dead giveaway makes it even less desirable to the average user.
> 
> This i can understand; however it's already incredibly undesirable to the average user. I'm not trying to make more or less people use it with this change, which is why i've left the core item the same; i want it to have a unique downside, instead of just an incredibly long list of upsides after you've finished a fetch quest.

Then it doesn't need the TC nerf.

**Comment by sylvia-from-fulp-station** [comment]
> Then it doesn't need the TC nerf.

I don't really give a shit about this PR anymore it's been marked DNM for 6 hours and i honestly don't like the final changes
